apps/prairielearn/assets/scripts/instructorAssessmentInstancesClient.ts

```diff
@@ -492,6 +491,7 @@ onDocumentReady(() => {
           class="btn btn-secondary btn-xs ml-1 time-limit-edit-button"
           id="row${row.assessment_instance_id}PopoverTimeLimit"
           data-row="${JSON.stringify(row)}"
+          data-placement="right"
```

Comment:
There usually isn't enough space on the right for me for this popover. Maybe bottom? Or use [fallback-placement](https://getbootstrap.com/docs/4.0/components/popovers/#options)?

Reply:
Can you show a screenshot of "not enough space on the right"? On my (admittedly pretty widescreen) monitor, there is, and even if there isn't, it'll automatically flip to the other side. I'm trying to understand what problem the current configuration causes.

Reply:
I was testing on an assessment with groups, and in that case the "Fingerprint changes" columns doesn't show. You should be able to see the same thing if you remove the column in the "Columns" button. In any case, I think fallback-placement may be a good idea in this context.

Reply:
I'd prefer bottom than left because left makes it show on top of the current remaining time, so editing becomes harder.

Reply:
My request for a screenshot stands. I don't know what you mean by "the Fingerprint changes columns doesn't show". Do you mean the popover covers that column?

Reply:
![image](https://github.com/user-attachments/assets/ec70060f-3dba-41b1-ab03-25590d4e5c8c)
![image](https://github.com/user-attachments/assets/36ab46ba-5adb-4bd1-b430-26b9134ca706)


> I don't know what you mean by "the Fingerprint changes columns doesn't show". Do you mean the popover covers that column?

I mean that the fingerprint column is not rendered for group assessments.

Reply:
Ah, I see - the issue is that the popover covers the current value in the column? I can change this for sure.

Reply:
This is easily fixed by setting `boundary: 'window'` in the settings for this popover.

Reply:
🤦 I always forget about that. Done in 1178b1c.